docker/Dockerfile-ubuntu-20.04

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04 as base
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" \
```

Comment:
UTC time zone is better and applicable universally and/or passing in a time zone would also be good.

Reply:
Yes, we can pass an environment variable into Docker using --env option.

Reply:
The PR has been updated to require --build-arg TIME_ZONE during docker build